src/sgame/sg_bot.cpp

```diff
@@ -396,6 +400,18 @@ Bot Thinks
 =======================
 */
 
+static void SetPing( gentity_t *self )
+{
+	if ( navMeshLoaded == navMeshStatus_t::GENERATING )
+	{
+		self->client->ps.ping = 998;
+	}
+	else
+	{
+		self->client->ps.ping = self->botMind->botSkill.level;
```

Comment:
I prefer if you move “bot skill as bot ping” hack in another PR as I don't want to block the merging of the navmesh generation feature because of some talk on this topic.

I'm fine with “ping 998” hack though, the ping explains why the bot doesn't respond, so the reusing of that value isn't bad, and even third party software can assume the player is unavailable even without knowing what we exactly mean.